packages/stockflux-launcher/src/helpers/getUndockedPosition.js

```diff
@@ -0,0 +1,102 @@
+import { OpenfinApiHelpers } from 'stockflux-core';
+
+async function getBounds() {
+  let bounds = await OpenfinApiHelpers.getCurrentWindowSync().getBounds();
+  return bounds;
+}
+
+async function getMonitorInfo() {
+  let monitorInfo = await window.fin.System.getMonitorInfo();
+  return monitorInfo;
+}
+
+function boundsCheck(bounds, screen, primaryCheck) {
+  if (primaryCheck) {
+    const lInBounds = bounds.left >= screen.left && bounds.left < screen.right;
+    const tInBounds = bounds.top >= screen.top && bounds.top < screen.bottom;
```

Comment:
Do these mean LeftInBounds, TopInBounds, RightInBounds?

Reply:
Updated variable names for clarity